packages/update-monitor/src/projects/synapse/contracts/synapseBridge.ts

```diff
@@ -0,0 +1,52 @@
+import { providers } from 'ethers'
+
+import {
+  getEip1967Admin,
+  getEip1967Implementation,
+} from '../../../common/eip1967'
+import { SynapseBridge__factory } from '../../../typechain/factories/synapse'
+import { ContractParameters } from '../../../types'
+import { addresses } from '../constants'
+
+export async function getSynapseBridge(
+  provider: providers.JsonRpcProvider,
+): Promise<ContractParameters> {
+  const synapseBridge = SynapseBridge__factory.connect(
+    addresses.synapseBridge,
+    provider,
+  )
+
+  const ADMIN_ROLE = await synapseBridge.DEFAULT_ADMIN_ROLE()
+  const GOVERNANCE_ROLE = await synapseBridge.GOVERNANCE_ROLE()
+  const NODEGROUP_ROLE = await synapseBridge.NODEGROUP_ROLE()
```

Comment:
One other idea is since the Role pattern from OpenZeppelin is quite common, maybe we can have some common utils to get all the members?